apricot/local/service.go

```diff
@@ -481,13 +482,16 @@ func (s *Service) GetRuntimeEntries(component string) (map[string]string, error)
 	s.logMethod()
 
 	if keys, err := s.ListRuntimeEntries(component); err == nil {
+		var keysErrors *multierror.Error
 		entries := make(map[string]string)
 		for _, key := range keys {
 			if entry, err := s.GetRuntimeEntry(component, key); err == nil {
 				entries[key] = entry
+			} else {
+				keysErrors = multierror.Append(keysErrors)
```

Comment:
You're not appending anything into your keysErrors here, otherwise the idea is right.